dev-util/envman/envman-1.2.1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-build
+
+EGO_ON="github.com/bitrise-io"
+EGO_PN="${EGO_ON}/${PN}"
+
+DESCRIPTION="Environment variable manager for Bitrise CLI."
```

Comment:
I almost merged this, then noticed you've used a dot when there's no proper sentence ;-). But I'll just fix that and merge.

Reply:
Thanks I'll remember that on next update.